docs/things/database.html

```diff
@@ -33,99 +33,6 @@ <h2>API Documentation</h2>
                         <li>
                                 <a class="function" href="#Database.__init__">Database</a>
                         </li>
-                        <li>
```

Comment:
no need to push changes on the documentation, as our friendly GitHub Actions bot will do this for us

Reply:
@AlexanderWillner 
I has to fix because of a merge conflict. This is what happened:

1. I pushed without updating HTML documentation
2. In my local github.com/mikez branch the GitHub Actions got triggered
3. I did a Pull Request.
4. Github told me the docs/things/database.html has a merge conflict.
5. I had to fix manually on github.com